src/modules/overview/actions/fetchNFTItems.ts

```diff
@@ -129,7 +129,11 @@ export const fetchNFTItems = createSmartAction<
 
           const tradePools = poolsData.data
             .filter(item => item.state !== 1)
-            .map(mapNFTItem);
+            .map(mapNFTItem)
+            .map((item: INFTItem) => ({
+              ...item,
+              tokenSymbol: poolsData?.tokenSymbol ?? TokenSymbol.BNB,
+            }));
```

Comment:
Can the above two loops be combined into one to reduce the program calculation time, what do you think?